Prox/Prox/PlaceDetails/PlaceDetailsIconInfoView.swift

```diff
@@ -70,18 +79,33 @@ class PlaceDetailsIconInfoView: UIView {
     }
 
     private func setupLabelContainerSubviews() -> [NSLayoutConstraint] {
-        labelContainer.addSubview(primaryTextLabel)
-        var constraints = [primaryTextLabel.topAnchor.constraint(equalTo: labelContainer.topAnchor),
-                           primaryTextLabel.leadingAnchor.constraint(equalTo: labelContainer.leadingAnchor),
-                           primaryTextLabel.trailingAnchor.constraint(equalTo: labelContainer.trailingAnchor),
-                           primaryTextLabel.heightAnchor.constraint(equalTo: labelContainer.heightAnchor, multiplier: 0.5)]
+        secondaryTextLabelHalfHeightConstraint = secondaryTextLabel.heightAnchor.constraint(equalTo: labelContainer.heightAnchor, multiplier: 0.5)
```

Comment:
If you put the 2 different multipliers as constants then you can you just refer to the constants when setting them